src/components/views/settings/tabs/user/HelpUserSettingsTab.js

```diff
@@ -256,21 +256,20 @@ export default class HelpUserSettingsTab extends React.Component {
                     <span className='mx_SettingsTab_subheading'>{_t("Versions")}</span>
                     <div className='mx_SettingsTab_subsectionText'>
                         {_t("%(brand)s version:", { brand })} {appVersion}<br />
-                        {!SettingsStore.getValue(UIFeature.ShowSimplifiedVersionInformation) && <>
+                        {SettingsStore.getValue(UIFeature.ShowSimplifiedVersionInformation) && <>
                             {_t("olm version:")} {olmVersion}<br />
                             {updateButton}
                         </>}
                     </div>
                 </div>
                 {this._renderLegal()}
                 {this._renderCredits()}
-                {SettingsStore.getValue(UIFeature.ShowAdvancedAboutInformation) &&
-                    <div className='mx_SettingsTab_section mx_HelpUserSettingsTab_versions'>
+                {<div className='mx_SettingsTab_section mx_HelpUserSettingsTab_versions'>
                         <span className='mx_SettingsTab_subheading'>{_t("Advanced")}</span>
                         <div className='mx_SettingsTab_subsectionText'>
                             {_t("Homeserver is")} <code>{MatrixClientPeg.get().getHomeserverUrl()}</code><br />
```

Comment:
In your comment on the user story you said:
Clear media cache is controlled by ShowAdvancedAboutInformation flag which also hides homserver, identity server and access token detail not been mentioned in this user story anywhere and were available in web
But with your change the homeserver and identity server are always shown?